rubocop.yml

```diff
@@ -2434,25 +2434,6 @@ Style/BlockDelimiters:
   #   collection.each do |element| puts element end
   AllowBracesOnProceduralOneLiners: false
 
-Style/BracesAroundHashParameters:
-  Description: 'Enforce braces style around hash parameters.'
-  Enabled: true
```

Comment:
Seems good,
but for future documentation of what rules exists and what is enabled/disabled, and not to clash with any future default changes in rubocop, we just change the `Enabled` flag accordingly instead of removing it all

Reply:
It errors out for the rule not being available any more, it's not that it's disabled.

Reply:
```
➜  pubsub git:(enh/upgrade-gem) ✗ rubocop -a -x .
Error: The `Style/BracesAroundHashParameters` cop has been removed.
(obsolete configuration found in .rubocop-https---raw-githubusercontent-com-TailorBrands-tailor-rubocop-master-rubocop-yml, please update it)
```

Reply:
ah, cool. ok then